python/tests/operations/drop_blocks.py

```diff
@@ -0,0 +1,131 @@
+import os
+
+import numpy as np
+
+import pytest
+import equistore
+from equistore import TensorMap, Labels
+
+
+DATA_ROOT = os.path.join(os.path.dirname(__file__), "..", "data")
+
```

Comment:
```suggestion
@pytest.fixture
def test_tensor_map() -> TensorMap:
    return equistore.io.load(
            os.path.join(DATA_ROOT, "qm7-power-spectrum.npz"),
            use_numpy=True,
        )
```

Reply:
Then for each function, add `test_tensor_map` as an argument. For instance:

```py
def drop_all(self, test_tensor_map):
    ...
```
and of course in the body of the functions the variable name should be changed from `tensor` to `test_tensor_map`.